src/caching.js

```diff
@@ -53,7 +53,16 @@
       if (typeof(path) !== 'string') {
         throw new Error('path should be a string');
       }
-      if (typeof(value) === 'undefined') {
+      if (!RemoteStorage.util.isFolder(path)) {
+        //throw new Error('path should be a folder');
```

Comment:
Alright done, though I still wonder why there are errors flashing across the screen and yet the tests pass.